web/src/views/analyze_sleep/waiting_for_server/index.js

```diff
@@ -11,27 +11,31 @@ const WaitingForServer = () => {
   }, [setAssets]);
 
   return (
-    <Container>
-      <h3 className="mb-4">Waiting for local server to be running...</h3>
+    <Container className="my-6">
+      <Alert color="warning" className="lead mb-5 mr-3">
+        <i class="fas fa-server fa-lg mr-3" />
+        <strong>Waiting for local server to be running...</strong>
```

Comment:
Maybe add something like this to the text?

Reply:
Like what :interrobang:

Reply:
oops bad copy pasta

here's the link: https://stackoverflow.com/questions/4639716/dot-dotdot-dotdotdot-as-loading